system-int-test/test/cmr/system_int_test/search/subscription/subscription_search_test.clj

```diff
@@ -205,6 +205,32 @@
             (search/find-concepts-in-format
              nil :subscription {} {:url-extension "atom"}))))))
 
+(deftest search-for-subscription-type-field
```

Comment:
This test can be removed. The `type` field in JSON response should be verified in `cmr.system-int-test.utils.subscription-util/assert-subscription-search` by adding `:type` to `json-field-names`.